generators/app/index.js

```diff
@@ -1,114 +1,109 @@
-var generators = require('yeoman-generator');
-var _ = require('lodash');
+var generators = require('yeoman-generator')
+var _ = require('lodash')
 
 module.exports = generators.Base.extend({
 
   _paths: function () {
-    this.log( this.destinationRoot() );
-    this.log( this.destinationPath( this.classname + '.js') );
-    this.log( this.sourceRoot() );
-    this.log( this.templatePath('index.js') );
+    this.log(this.destinationRoot())
+    this.log(this.destinationPath(this.classname + '.js'))
+    this.log(this.sourceRoot())
+    this.log(this.templatePath('index.js'))
   },
 
   // The name `constructor` is important here
   constructor: function () {
     // Calling the super constructor is important so our generator is correctly set up
-    generators.Base.apply(this, arguments);
+    generators.Base.apply(this, arguments)
 
     // This makes `classname` a required argument.
-    this.argument('classname', { type: String, required: true, desc: 'Class name' });
+    this.argument('classname', { type: String, required: true, desc: 'Class name' })
     // And you can then access it later on this way; e.g. CamelCased
-    this.classname = _.camelCase(this.classname);
+    this.classname = _.camelCase(this.classname)
 
-
-    this.attrs = [];
+    this.attrs = []
 
     // Next, add your custom code
-    this.option('coffee'); // This method adds support for a `--coffee` flag
-
-    this.spawnCommand('ls', ['-la']);
-
+    this.option('coffee') // This method adds support for a `--coffee` flag
   },
 
-  _method: function() {
-    this.log('won\'t be called automatically');
-    this.log('Cause method name is prefixed by an underscore (e.g. _method).');
+  _method: function () {
+    this.log('won\'t be called automatically')
+    this.log('Cause method name is prefixed by an underscore (e.g. _method).')
   },
 
-  askForAttribute: function() {
-    var done = this.async();
-    this.prompt( {
-      type    : 'confirm',
-      name    : 'newAttribute',
-      message : 'Do you wanna add new attribute',
-      default : true
+  askForAttribute: function () {
```

Comment:
promptForUserInput. 

And mark this methods as @private or @public accordingly.JSDOC